src/Ninja/G1CP/Content/Tests/test015.d

```diff
@@ -60,5 +53,5 @@ func int G1CP_Test_015() {
     hero.attribute[ATR_STRENGTH] = strengthBak;
 
     // At last
-    return passed;
+    return result;
```

Comment:
I like the variable name `passed` here better actually. (I think I had suggested the name "result", but "passed" is more descriptive.)